salt/modules/zcbuildout.py

```diff
@@ -511,7 +499,7 @@ def upgrade_bootstrap(
     else:
         buildout_ver = _get_buildout_ver(directory)
         booturl = _get_bootstrap_url(directory)
-    LOG.debug("Using {0}".format(booturl))  # pylint: disable=str-format-in-logging
+    LOG.debug("Using {}".format(booturl))  # pylint: disable=str-format-in-logging
```

Comment:
```suggestion
    LOG.debug("Using %s", booturl)
```

Reply:
same as the others in this module, we need to make the LOG object a real logger and unwind whatever they are trying to do.